releases/development/master/extra/barclamp_install.rb

```diff
@@ -109,15 +120,28 @@ def rm_tmpdir(tmpdir)
     puts "Barclamp at #{bc} has no name, skipping"
     next
   end
+  if barclamp["barclamp"] and barclamp["barclamp"]["version"] and \
+    barclamp["barclamp"]["version"].to_i 
```

Comment:
> Btw, we need a else in case the key is missing, don't we?

Sorry, which key? Missing "barclamp" will be caught by rescue, missing "version" will be nil, converted to 0 by to_i.